spot-ml/src/main/scala/org/apache/spot/SuspiciousConnects.scala

```diff
@@ -43,16 +44,22 @@ object SuspiciousConnects {
         val sqlContext = new SQLContext(sparkContext)
         implicit val outputDelimiter = config.outputDelimiter
 
+        val inputDataFrame = InputOutputDataHandler.getInputDataFrame(sqlContext, config.inputPath, logger)
+          .getOrElse(sqlContext.emptyDataFrame)
+        if(inputDataFrame.rdd.isEmpty()) System.exit(0)
```

Comment:
I don't think it's a bad design, I'd just like a message printed on exit explaining what has happened